PayPalNativeCheckout/src/main/java/com/paypal/android/checkout/PayPalCheckoutClient.kt

```diff
@@ -2,20 +2,23 @@ package com.paypal.android.checkout
 
 import android.app.Application
 import com.paypal.android.core.API
-import com.paypal.android.checkout.model.buyer.Buyer
 import com.paypal.android.core.APIClientError
 import com.paypal.android.core.CoreConfig
+import com.paypal.android.core.PayPalSDKError
 import com.paypal.checkout.PayPalCheckout
 import com.paypal.checkout.approve.OnApprove
 import com.paypal.checkout.cancel.OnCancel
 import com.paypal.checkout.config.CheckoutConfig
 import com.paypal.checkout.createorder.CreateOrder
 import com.paypal.checkout.error.OnError
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
 
 /**
  * Use this client to checkout with PayPal.
  */
-class PayPalClient internal constructor (
+class PayPalCheckoutClient internal constructor (
```

Comment:
I can't fully remember, but I _think_ the logic behind calling it `PayPalClient` is to make it stand out as the default integration pattern for `PayPal`. We would have to update the module name as well though to be `:PayPal` instead of `:PayPalNativeCheckout`.

Reply:
okay, going to revert the client name. But the module name im going to leave it for a different PR, i dont want git messing around and suddenly have like a 1 million file change